app/(tabs)/transportation/status.tsx

```diff
@@ -57,12 +57,17 @@ export const StatusScreenDetails = ({ day, groupedUsers }) => {
     { key: '0', value: 'Looking for Walking Buddy' },
     { key: '1', value: 'Looking for CTA/Shuttle Buddy' },
     { key: '2', value: 'Looking for Carpool' },
-    { key: '3', value: 'Can Drive People' },
+    { key: '3', value: 'Willing to Drive' },
   ];
 
+  const totalVolunteers = Object.values(groupedUsers).reduce<number>(
+    (sum, array) => sum + (array as any[]).length,
```

Comment:
Minor: can we use UserInfo[] instead of any[]. We generally don't use any to type for typescript